src/pages/partners/affiliate-ib/_deriv-ib-programme.tsx

```diff
@@ -131,7 +134,62 @@ const SubtitleHeader = styled(Header)`
     }
 `
 
+type ItemProps = {
+    active_tab: 'mt5' | 'x'
+    name: 'mt5' | 'x'
+}
+
+const Item = styled.div<ItemProps>`
+    margin-top: 4rem;
+    padding: 1.2rem 1.6rem;
+    opacity: ${(props) => (props.active_tab === props.name.toLocaleLowerCase() ? '1' : '0.48')};
+    border-bottom: ${(props) =>
+        props.name === props.active_tab
+            ? '2px solid var(--color-red)'
+            : '2px solid var(--color-black-3)'};
+    cursor: pointer;
+    z-index: 2;
+    white-space: nowrap;
+
+    ${Header} {
+        font-size: 2.4rem;
+        width: max-content;
+        text-align: center;
+    }
+    h4 {
+        color: ${(props) =>
+            props.active_tab === props.name.toLocaleLowerCase()
+                ? 'var(--color-red)'
+                : 'var(--color-black-3)'};
+        font-weight: ${(props) =>
+            props.active_tab === props.name.toLocaleLowerCase() ? '300' : '400'};
+    }
+    @media ${device.tabletL} {
+        padding: 12px 8px 12px 7px;
+        margin-top: 24px;
```

Comment:
Hi @eduard-deriv  You can use `rem` instead of `px`..we are using rem in all places